ompi/include/mpi.h.in

```diff
@@ -193,6 +193,9 @@
 /* Whether C compiler supports -fvisibility */
 #undef OPAL_C_HAVE_VISIBILITY
 
+/* Whether OMPI shoud define ULFM Fault Tolerant constants */
+#undef OPAL_ENABLE_FT_MPI
```

Comment:
This makes me super nervous.  It seems like it would be much better to just always define the public bits of the FT library than to change the ABI based on whether or not FT is built?